nixos/hosts/lab/sshfs.nix

```diff
@@ -58,6 +58,12 @@
     };
   };
 
+  # Ensure that paths exist
+  systemd.services."docker-compose-runner".unitConfig = {
```

Comment:
is there a way to "wait" ? does sshfs create this folder ? we need to investigate more what happens when offline